roles/kubernetes-apps/ansible/templates/coredns-config.yml.j2

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: coredns
+  namespace: {{ system_namespace }}
+data:
+  Corefile: |
+    .:53 {
+        errors
+        health
+        kubernetes {{ cluster_name }} in-addr.arpa ip6.arpa {
```

Comment:
it should only resolve reverse ipv6 adresses ? or there is no need to add ipv4 arpa ?

Reply:
Dont know yet. This is just de default config that it uses from the link, we tested it and worked perfect. ;)

Reply:
ok, then it looks good.